src/Middleware/Session.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Had to bring this in from a package we were using which didn't have the middleware upgraded for Slim 4.